mixins/timeUtils.ts

```diff
@@ -19,7 +19,7 @@ declare module '@nuxt/types' {
 
 declare module 'vue/types/vue' {
   interface Vue {
-    ticksToMs: (ticks: number) => number;
+    ticksToMs: (ticks: number | null | undefined) => number;
```

Comment:
The behaviour of this whole file can't be accomplished with date fns? What does ticks equals to? Never understood what it meant in web.

Reply:
Ticks is something from C#. It's basically the unit of representation of time over there.

The file is meant to convert ticks sent by the server (There are plans to change this in 11.0, I believe) to milliseconds in one central place to avoid all the undocumented `thing * 1000` and `thing / 1000` all over jf-web.